pkg/data/components/HoneycombExporter.yaml

```diff
@@ -54,7 +54,6 @@ properties:
     type: string
     validations:
       - noblanks
-      - url
```

Comment:
We could update the url validator to optionally allow a port.

Reply:
Now https://github.com/honeycombio/hpsf/commit/af0c25b487cb45afd5483fee60d4a3c448cd09f1 is merged. We should be okay to keep this validator.